main.py

```diff
@@ -55,15 +55,22 @@ def openfile(self):
 
     def double_click_search_table(self, info):
         file_path = self.resultTable.item(info.row(), 0).text()
-        os.startfile(os.path.normpath(file_path))
+        try:
+            os.startfile(file_path)

```

Comment:
可以使用 os.path.exists() 方法检查文件是否存在，避免宽泛的 try-except。

Reply:
已修改对打开图片文件的校验。

---

标记删除处参考 `hnswlib` 库的文档：

> `mark_deleted(label)` - marks the element as deleted, so it will be omitted from search results. Throws an exception if it is already deleted.

似乎只能由 `try...except` 接收。